### DISCUSSION TITLE
Is there a way to run redocusaurus only?

### DISCUSSION THREAD

**Discussion by SVilgelm** [rohit-gohri/redocusaurus#243] [Q&A] [answered]
Hello,

we use `docusaurus` v2.1.0 and `redocusaurus` v1.4.0. As a developer of the API I need to check the API only and don't care about the main documentation handled by docusaurus.
So my question is simple. Is there a simple way to run the redocusaurus only without compiling whole documentation?
Right now  I use `docusaurus start` to build the documentation, which opens a browser on main page, then I click on API and open redoc. So I want to skip the documentation step :)

Thank you!

**Comment by MarcLevisse** [comment, marked as answer]
Why not using only Redocly's openapi-cli ?
Simple way to "display" your api document through ReDoc

```bash
npx @redocly/openapi-cli preview-docs --port=8081 yourspec.yaml
```

no build time, live reload

**Reply by SVilgelm** [reply]
Thank you! I did not know about that tool.

I checked and found that the `@redocly/openapi-cli` is deprecated, the correct package is `@redocly/cli`

The issue is solved

**Comment by rohit-gohri** [comment]
You can set the route as `/` in redocusaurus config and then it should render that as home page. Remove the homepage `jsx/tsx` file you might have in the `src` folder or they might conflict. 

If you only have 1 single api schema and no separate docs then you can also do what @MarcLevisse suggested and directly use [redoc](https://github.com/redocly/redoc/), that's what redocusaurus uses under the hood. But if you have separate markdown docs or multiple APIs then I think redocusaurus is a better solution.

**Reply by MarcLevisse** [reply]
anyway, we @ActitoHQ use redocusaurus with happiness and are grateful for your great work @rohit-gohri !
